tidb-cloud/import-parquet-files.md

```diff
@@ -83,71 +83,71 @@ Because Parquet files do not contain schema information, before importing data f
     - **Data Format**: select **Parquet**.
-    - **Setup Credentials** (This field is visible only for AWS S3): enter the Role ARN value for **Role-ARN**.
-    - **Target Cluster**: fill in the **Username** and **Password** fields.
-    - **DB/Tables Filter**: if you want to filter which tables to be imported, you can specify one or more table filters in this field, separated by `,`.
-
-        For example:
+    - **Location**: select the location where your Parquet files are located.
+    - **Bucket URL**: select the bucket URL where your Parquet files are located.
+    - **Role-ARN**: (This field is visible only for AWS S3): enter the Role ARN value for **Role-ARN**.
+    - **Target Cluster**: shows the cluster name and the region name.
 
-        - `db01.*`: all tables in the `db01` database will be imported.
-        - `db01.table01*,db01.table02*`: all tables starting with `table01` and `table02` in the `db01` database will be imported.
-        - `!db02.*`: except the tables in the `db02` database, all other tables will be imported. `!` is used to exclude tables that do not need to be imported.
-        - `*.*` : all tables will be imported.
+    Confirm the compliance of cross region. Click **Next**.
 
-        For more information, see [table filter snytax](/table-filter.md#syntax).
+4. Modify the file patterns and add the table filter rules if needed.
 
-    - **Custom Pattern**: enable the **Custom Pattern** feature if you want to import Parquet files whose filenames match a certain pattern to a single target table.
+    - **File Pattern**: modify the file pattern if you want to import Parquet files whose filenames match a certain pattern to a single target table.
 
         > **Note:**
         >
-        > After enabling this feature, one import task can only import data to a single table at a time. If you want to use this feature to import data into different tables, you need to import several times, each time specifying a different target table.
+        > When you use this feature, one import task can only import data to a single table at a time. If you want to use this feature to import data into different tables, you need to import several times, each time specifying a different target table.
 
-        When **Custom Pattern** is enabled, you are required to specify a custom mapping rule between Parquet files and a single target table in the following fields:
+        When you use this feature, you need to specify a custom mapping rule between Parquet files and a single target table in the following fields:
 
-        - **Object Name Pattern**: enter a pattern that matches the names of the Parquet files to be imported. If you have one Parquet file only, you can enter the filename here directly.
+        - **File Name**: enter a pattern that matches the names of the Parquet files to be imported. If you have one Parquet file only, you can enter the filename here directly.
```

Comment:
```suggestion
        - **Source file name**: enter a pattern that matches the names of the Parquet files to be imported. If you have one Parquet file only, you can enter the filename here directly.
```

Reply:
the parquet file name should be ended with the suffix of ".parquet"

Reply:
OK added a note to remind users.